packages/fx-core/resource/package.nls.json

```diff
@@ -587,7 +587,7 @@
   "core.getUserEmailQuestion.validation1": "Email address cannot be null or empty",
   "core.getUserEmailQuestion.validation2": "Please change [UserName] to the real user name",
   "core.collaboration.error.failedToLoadDotEnvFile": "Unable to load your .env File. Reason: %s",
-  "core.collaboration.error.failedToGetTeamsAppId": "Unable to find Teams app id in your .env file. Please correctly set the value with key '%s' in your .env file.",
+  "core.collaboration.error.failedToGetTeamsAppId": "Unable to find Teams app id in your .env file. Make sure you have already run 'Provision' or correctly set the value with key '%s' in your .env file.",
```

Comment:
can you reuse existing error type: UnresolvedPlaceholderError

"error.common.UnresolvedPlaceholderError": "The program cannot proceed as the following environment variables are missing: '%s', which are required for file: %s."